api/module.mk

```diff
@@ -28,6 +28,9 @@ TEST_SCM_SRC       = $(shell $(FIND) $(TEST_DIR) -type f -name "*.$(CLJ)")
 JAR          := $(DERIVED_TARGET_DIR)/wfl-$(WFL_VERSION).jar
 JAR_LINK     := $(DERIVED_TARGET_DIR)/wfl.jar
 
+SYSTEM_TESTS := \
+	test-append-to-aou-workload
```

Comment:
I don't think the makefile should have these details and i'm not sure implementing the parallelism in the makefile is right either

Reply:
Fair enough--I think the clojure route might be a bit more extensible